pkg/diskmaker/controllers/lv/diskmaker.go

```diff
@@ -255,22 +262,35 @@ func (d *DiskMaker) symLinkDisks(diskConfig *DiskConfig) {
 			}
 			msg = strings.Join(diff.List(), ", ") + " was defined in devicePaths, but expected a path in /dev/"
 		}
-		e := NewEvent(ErrorFindingMatchingDisk, msg, "")
-		d.eventSync.Report(e, d.localVolume)
+		r.eventSync.Report(r.localVolume, newDiskEvent(ErrorFindingMatchingDisk, msg, "", corev1.EventTypeWarning))
 		klog.Errorf(msg)
-		return
+		return reconcile.Result{}, nil
 	}
 
 	for storageClass, deviceArray := range deviceMap {
 		for _, deviceNameLocation := range deviceArray {
-			symLinkDirPath := path.Join(d.symlinkLocation, storageClass)
-			d.createSymLink(deviceNameLocation, symLinkDirPath)
+			symLinkDirPath := path.Join(r.symlinkLocation, storageClass)
+			r.createSymLink(deviceNameLocation, symLinkDirPath)
 		}
 	}
 
+	return reconcile.Result{Requeue: true, RequeueAfter: checkDuration}, nil
+}
+
+func nodeSelectorMatchesNodeLabels(node *corev1.Node, nodeSelector *corev1.NodeSelector) (bool, error) {
```

Comment:
since this is in both lv and lvset diskmaker controllers, can we move this to the `common` pkg?